web/src/__tests__/ingestion.servertest.ts

```diff
@@ -1654,121 +1603,6 @@ IB Home   /   . . .   /   News   /   News about the IB   /   Why ChatGPT is an o
     expect(dbScore?.value).toBe(scoreValue);
   });
 
-  it("should error on Bearer Auth for a trace from different project", async () => {
```

Comment:
scoreprocessor does not check for this anymore. with the upcoming change of tracids being unique within a project, testing for this issue does not really make sense